website/docs/r/linux_virtual_machine_scale_set.html.markdown

```diff
@@ -108,35 +106,35 @@ The following arguments are supported:
 
 * `network_interface` - (Required) One or more `network_interface` blocks as defined below.
 
-* `os_disk` - (Required) An `os_disk` block as defined below.
+* `os_disk` - (Required) A `os_disk` block as defined below.
 
 ---
 
 * `additional_capabilities` - (Optional) A `additional_capabilities` block as defined below.
 
 * `admin_password` - (Optional) The Password which should be used for the local-administrator on this Virtual Machine. Changing this forces a new resource to be created.
 
--> **NOTE:** When an `admin_password` is specified `disable_password_authentication` must be set to `false`.
+-> **NOTE:** When a `admin_password` is specified `disable_password_authentication` must be set to `false`.
```

Comment:
an is the correct article to use here?

Reply:
I think so, if the first letter makes a vowel-type sound, you use "an"; if the first letter would make a consonant-type sound, you use "a." However, there are some exceptions to these rules.

Reply:
Fixed.